Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
I've added a test, hope it is what you want:
https://github.com/SpacePossum/PHP-CS-Fixer/commit/7d849f39e0e60ec3b5612869bd30c484848b6c66

(reverted the other changes in the next commit)